chromium/background.js

```diff
@@ -2,6 +2,28 @@
 
 (function(exports) {
 
+var all_rules, ls;
+
+try{
+  ls = localStorage;
+} catch(e) {
+  ls = {setItem: () => {}, getItem: () => {}};
```

Comment:
Any idea what this `try`/`catch` is for? When is `localStorage` not defined?

Reply:
I think so. But I think there are cases when it is defined, but trying to access the contents results in an error or are ignored (if localStorage is disabled).

Reply:
Attempting to access `localStorage` in Firefox when cookies are disabled results in an exception: `SecurityError: The operation is insecure.`